content/applications/inventory_and_mrp/inventory/management/reporting/using_inventory_valuation.rst

```diff
@@ -10,98 +10,94 @@ In Odoo, this process can be conducted manually— by warehouse employees physic
 products— or automatically through the database.
 
 .. seealso::
-   :ref:`Inventory valuation configuration <inventory/inventory_valuation_config>`
+   :ref:`Inventory valuation methods <inventory/management/reporting/inventory_valuation_config>`
 
-Understand the basics of inventory valuation
-============================================
+Inventory valuation basics
+==========================
 
 In order to understand how moving products in and out of stock affects the company's overall value,
 consider the following product and stock moves scenario below.
 
 Receive a product
 -----------------
 
-For example, consider an physical product, a simple *table*, which is categorized as `Office
+For example, consider a physical product, a simple *table*, which is categorized as `Office
 Furniture` in the product form's :guilabel:`Product Category` field.
 
 Navigate to the the product category itself by going to :menuselection:`Inventory app -->
-Configuration --> Product Categories`, and on the form, set the :guilabel:`Costing Method` as `First
-In First Out (FIFO)` and the :guilabel:`Inventory Valuation` field as `Automated`.
+Configuration --> Product Categories`. On the form, set the :guilabel:`Costing Method` as `First In
+First Out (FIFO)` and :guilabel:`Inventory Valuation` as `Automated`.
```

Comment:
If the field values "First In First Out" and "Automated" are _selects_ from a dropdown, then they should have :guilabel:s, otherwise if they are user input keep them as literal formatting 🙂
```suggestion
Configuration --> Product Categories`. On the form, set the :guilabel:`Costing Method` as :guilabel:`First In
First Out (FIFO)` and :guilabel:`Inventory Valuation` as :guilabel:`Automated`.
```